pyfolio/plotting.py

```diff
@@ -619,7 +552,7 @@ def show_perf_stats(returns, factor_returns, positions=None,
     returns : pd.Series
         Daily returns of the strategy, noncumulative.
          - See full explanation in tears.create_full_tear_sheet.
-    factor_returns : pd.Series
+    factor_returns : pd.Series, optional
         Daily noncumulative returns of the benchmark.
```

Comment:
In my mind, there isn't much ambiguity in this docstring: it's the daily returns of some benchmark factor (possibly risk factor). E.g. it could the returns of SPY, or the returns of any of the Fama French factors, etc.

Perhaps "benchmark factor" would be more explicit? I've changed the docstring to reflect that.

Reply:
With some searching, I'm realizing that the same parameter has very spotty docstrings in different functions. I'm changing all functions to be this same docstring:

```
    factor_returns : pd.Series
         Daily noncumulative returns of the benchmark factor to which betas are
         computed. Usually a benchmark such as market returns.
         - This is in the same style as returns.
```

Reply:
That docstring is definitely an improvement. 

My general objection to this parameter name is that `factor_returns` has ambiguous plurality. Is it expected to be the returns of a single factor (in which case, what factor is it?), or is it a dataframe of factor returns ala the Q risk model? Your updated docstring answers this question for me, but ideally we would have a name here that was more self-describing to begin with. (Changing the name shouldn't be in the scope of this PR though).